Sources/MongoSwift/ConnectionPool.swift

```diff
@@ -36,7 +36,7 @@ internal class ConnectionPool {
     }
 
     /// Initializes the pool in pooled mode using the provided `ConnectionString`.
-    internal init(from connString: ConnectionString) throws {
+    internal init(from connString: ConnectionString, withOptions options: TLSOptions? = nil) throws {
```

Comment:
i think the `withOptions` label can be omitted for consistency with how we pass our other options types.